curriculum/challenges/english/10-coding-interview-prep/take-home-projects/build-a-nightlife-coordination-app.english.md

```diff
@@ -9,15 +9,15 @@ forumTopicId: 302351
 
 ## Description
 <section id='description'>
-<strong>Objective:</strong> Build a full stack JavaScript app that is functionally similar to this: <a href='http://whatsgoinontonight.herokuapp.com/' target='_blank'>http://whatsgoinontonight.herokuapp.com/</a> and deploy it to Heroku.
+<strong>Objective:</strong> Build a full stack JavaScript app that is functionally similar to this: <a href='http://whatsgoinontonight.herokuapp.com/' target='_blank'>http://whatsgoinontonight.herokuapp.com/</a>.
```

Comment:
We should always be linking to the `https` version of a site.  The herokuapp.com `http` links do not automatically forward to the `https` version, so we should specify the `https` version. 
```suggestion
<strong>Objective:</strong> Build a full stack JavaScript app that is functionally similar to this: <a href='https://whatsgoinontonight.herokuapp.com/' target='_blank'>https://whatsgoinontonight.herokuapp.com/</a>.
```